### PR TITLE
[4.0] Plugins using more namespaced classes Part 2

### PR DESCRIPTION
**Pull request by anuragteapot**
### Summary of Changes

Switch to the namespaced classes

### Testing Instructions

Simple code review

### CODE REVIEW

**Comment by anuragteapot** [comment]
@C-Lodder please review changes

File: plugins/editors-xtd/article/article.php [hunk 1]
```diff
@@ -35,15 +41,15 @@ class PlgButtonArticle extends JPlugin
 	 */
 	public function onDisplay($name)
 	{
-		$input = JFactory::getApplication()->input;
-		$user  = JFactory::getUser();
+		$input = Factory::getApplication()->input;
```
Comment (joomdonation) [inline plugins/editors-xtd/article/article.php]: Not related but this $input variable is not used at all. Could you please remove it?

**Comment by C-Lodder** [comment]
Everything here seems to look ok

File: plugins/editors-xtd/pagebreak/pagebreak.php [hunk 2]
```diff
@@ -35,15 +40,15 @@ class PlgButtonPagebreak extends JPlugin
 	 */
 	public function onDisplay($name)
 	{
-		$input = JFactory::getApplication()->input;
-		$user  = JFactory::getUser();
+		$input = Factory::getApplication()->input;
```
Comment (joomdonation) [inline plugins/editors-xtd/pagebreak/pagebreak.php]: This $input variable is not used at all, so please remove this line

File: plugins/fields/checkboxes/tmpl/checkboxes.php [hunk 3]
```diff
@@ -23,7 +25,7 @@
 {
 	if (in_array((string) $value, $fieldValue))
 	{
-		$texts[] = JText::_($name);
+		$texts[] = Text::_($name);
 	}
 }
```
Comment (joomdonation) [inline plugins/fields/checkboxes/tmpl/checkboxes.php:31]: Could you please check plugins/fields/checkboxes/checkboxes.php? **FieldsListPlugin** should be replaced with \Joomla\Component\Fields\Administrator\Plugin\FieldsListPlugin and the line JLoader::import('components.com_fields.libraries.fieldslistplugin', JPATH_ADMINISTRATOR); should be removed (same with how the file plugins/fields/color/color.php is coded

File: plugins/fields/imagelist/imagelist.php [hunk 4]
```diff
@@ -9,6 +9,8 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\Form\Form;
+
 JLoader::import('components.com_fields.libraries.fieldslistplugin', JPATH_ADMINISTRATOR);
```
Comment (joomdonation) [inline plugins/fields/imagelist/imagelist.php]: Same with earlier comment, this line should be removed and FieldsListPlugin should be replaced with \Joomla\Component\Fields\Administrator\Plugin\FieldsListPlugin

File: plugins/fields/list/tmpl/list.php [hunk 5]
```diff
@@ -23,7 +25,7 @@
 {
 	if (in_array((string) $value, $fieldValue))
 	{
-		$texts[] = JText::_($name);
+		$texts[] = Text::_($name);
 	}
 }
```
Comment (joomdonation) [inline plugins/fields/list/tmpl/list.php:31]: Not in this file but you should check code in the file plugins/fields/list/list.php:

1. Remove JLoader::import('components.com_fields.libraries.fieldslistplugin', JPATH_ADMINISTRATOR);
2. Replace FieldsListPlugin with \Joomla\Component\Fields\Administrator\Plugin\FieldsListPlugin

File: plugins/fields/radio/tmpl/radio.php [hunk 6]
```diff
@@ -23,7 +25,7 @@
 {
 	if (in_array((string) $optionValue, $value))
 	{
-		$texts[] = JText::_($optionText);
+		$texts[] = Text::_($optionText);
 	}
 }
```
Comment (joomdonation) [inline plugins/fields/radio/tmpl/radio.php:31]: Same with my other comments, check the file plugins/fields/radio/radio.php and make necessary change:

1. Remove JLoader::import('components.com_fields.libraries.fieldslistplugin', JPATH_ADMINISTRATOR);
2. Replace FieldsListPlugin with \Joomla\Component\Fields\Administrator\Plugin\FieldsListPlugin

File: plugins/fields/sql/sql.php [hunk 7]
```diff
@@ -9,6 +9,11 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\Factory;
+use Joomla\CMS\Form\Form;
+use Joomla\CMS\Access\Access;
+use Joomla\CMS\Language\Text;
+
 JLoader::import('components.com_fields.libraries.fieldslistplugin', JPATH_ADMINISTRATOR);
```
Comment (joomdonation) [inline plugins/fields/sql/sql.php]: Remove this line and and replace FieldsListPlugin with \Joomla\Component\Fields\Administrator\Plugin\FieldsListPlugin

**Comment by joomdonation** [comment]
There are still some small issues as I commented above. Please correct it and we should be fine. Also, I don't know if the change to filesystem plugin should be made here or on media manager repo

**Comment by anuragteapot** [comment]
@joomdonation yeah, I am removing changes to filesystem because this file is a part of media manager and they have a separate repo. What do you think?

Thanks

**Comment by joomdonation** [comment]
I am unsure. Maybe keep the change for now as I guess the change here is also synchronized to media manager repo, too. For now, please make the small changes which I commented and you should be done with this PR.

**Comment by anuragteapot** [comment]
Ok

File: plugins/fields/checkboxes/tmpl/checkboxes.php (hunk 3 above)
Comment (anuragteapot) [reply plugins/fields/checkboxes/tmpl/checkboxes.php:31]: yeah, we can also write in the form as you explain.

**Comment by anuragteapot** [comment]
@joomdonation done!

**Comment by anuragteapot** [comment]
Please review changes

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on aff2679b7f26a873a9277f0138fb0a24b544d471<br /><br />Great work, thanks @Anu1601CS<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19323">issues.joomla.org/tracker/joomla-cms/19323</a>.</sub>

**Comment by anuragteapot** [comment]
@joomdonation thanks, I contribute to joomla as I can.

**Comment by anuragteapot** [comment]
@C-Lodder if you tested then Mark result. Thanks

**Comment by anuragteapot** [comment]
reminder to @C-Lodder.

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on aff2679b7f26a873a9277f0138fb0a24b544d471<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19323">issues.joomla.org/tracker/joomla-cms/19323</a>.</sub>

**Pull request merged**